classes/submissionFile/maps/Schema.php

```diff
@@ -175,6 +175,13 @@ protected function mapByProperties(array $props, SubmissionFile $submissionFile)
                 continue;
             }
 
+            if ($prop === 'uploaderUserName') {
+                $user = Repo::user()->get($submissionFile->getData('uploaderUserId'));
+                $output[$prop] = $user ? $user->getUsername() : '';
```

Comment:
If `null` is ok here, then `$user?->getUsername()` or perhaps `$user?->getUsername() ?? ''`

Reply:
Thanks! I went with the second option, because I prefer the API to return consistent types (eg - always a string) when possible.